acceptancetests/assess_recovery.py

```diff
@@ -64,6 +64,15 @@ def enable_ha(bs_manager, controller_client):
         controller_client, bs_manager.known_hosts)
     bs_manager.known_hosts = remote_machines
 
+def disable_ha(bs_manager, controller_client):
+    """Disable HA and wait for the controllers to be ready."""
+    log.info("Disabling HA.")
+    controller_client.wait_for(controller_client.remove_machines(["1", "2"], controller=True))
+    show_controller(controller_client)
```

Comment:
what's the show_controller doing here?

Reply:
It's really helpful when debugging to see what state the controller is in. Same for enable_ha.

Reply:
maybe a pre-log line then that helps the log show that this is kicked off as log/tracking output?